daily.js

```diff
@@ -21,7 +21,7 @@ const dailyPreviously = () => {
     issuesByRange({ start, end })
       .then((issues) => issues.filter(ignoreProjects))
       .then((issues) => issues.filter(ignoreBotUsers))
-      .then((issues) => enrichIssues({ issues, start, end })),
+      .then((issues) => enrichIssues({ issues, start, end, skipEnrichPRs: false, skipEnrichComments: false })),
```

Comment:
don't you have to do the same on the `weekly.js` one since the default is always true?

Reply:
ye default is true and digests (like weekly) don't really need them, aside from the previous dumb decision to format whether it's PR/Issues based on `enriched_commits` existence which has been simplified along in this PR.